docs/reference/all_strategies.rst

```diff
@@ -8,6 +8,8 @@ Here are the docstrings of all the strategies in the library.
 
 .. automodule:: axelrod.strategies.adaptive
    :members:
+.. automodule:: axelrod.strategies.adaptivezerodet
+   :members:
```

Comment:
Currently the tests are failing because of the documentation:

```
Warning, treated as error:
406autodoc: failed to import module 'adaptivezerodet' from module 'axelrod.strategies'; the following exception was raised:
407No module named 'axelrod.strategies.adaptivezerodet'
408Makefile:53: recipe for target 'html' failed
409make: *** [html] Error 2
410The command "cd docs; make clean; make html" exited with 2.
```

I believe we can just delete these two lines:

```suggestion

```

Reply:
Done